src/fields.ts

```diff
@@ -88,6 +141,11 @@ export interface RTLabelNode extends RTSpanNodeBase {
 }
 
 // Media nodes
+
+/**
+ * Rich Text <img /> nodes. They could link to other documents, external web links and media fields
+ * @typeParam linkTo - A link to either a document, external site or media
```

Comment:
`@typeParam` refers to generic types, which this type does not have.

For example, this is what a type with a generic looks like:

```ts
type RTImageNode<LinkTo> {
  linkTo: LinkTo
}
```

It's like a function parameter, but for a type.

Since this type doesn't have a generic, we can remove the `@typeParam` line.